projects/bp-gallery/src/components/views/geoguessr/GeoMap.tsx

```diff
@@ -0,0 +1,210 @@
+import 'leaflet/dist/leaflet.css';
+import { MapContainer, TileLayer, useMapEvent, Marker, useMap } from 'react-leaflet';
+import { useEffect, useMemo, useRef, useState } from 'react';
+import { Button } from '@mui/material';
+import {
+  GetPictureGeoInfoQuery,
+  useCreatePictureGeoInfoMutation,
+  useIncreaseNotAPlaceCountMutation,
+} from '../../../graphql/APIConnector';
+import { LatLngBounds, Icon, Map } from 'leaflet';
+import { useTranslation } from 'react-i18next';
+
+const PlayerMarkers = ({
+  allGuesses,
+  myGuess,
+}: {
+  allGuesses: GetPictureGeoInfoQuery['pictureGeoInfos'];
+  myGuess?: { lat: number; lng: number };
+}) => {
+  const map = useMap();
+  const coords = useMemo(
+    () =>
+      allGuesses?.data.map(x => ({
+        lat: x.attributes!.latitude!,
+        lng: x.attributes!.longitude!,
+      })) ?? [],
+    [allGuesses]
+  ) as { lat: number; lng: number }[];
+
+  useEffect(() => {
+    setTimeout(() => {
+      map.invalidateSize();
+      coords.length !== 0 &&
+        map.flyToBounds(
+          new LatLngBounds(
+            {
+              lat: myGuess
+                ? Math.min(...coords.map(x => x.lat), myGuess.lat)
+                : Math.min(...coords.map(x => x.lat)),
+              lng: myGuess
+                ? Math.min(...coords.map(x => x.lng), myGuess.lng)
+                : Math.min(...coords.map(x => x.lng)),
+            },
+            {
+              lat: myGuess
+                ? Math.max(...coords.map(x => x.lat), myGuess.lat)
+                : Math.max(...coords.map(x => x.lat)),
+              lng: myGuess
+                ? Math.max(...coords.map(x => x.lng), myGuess.lng)
+                : Math.max(...coords.map(x => x.lng)),
+            }
+          )
+        );
+    }, 300);
+  }, [coords, myGuess, map]);
+  const othersIcon = new Icon({
+    iconUrl: '/images/location-map-pin.svg',
+    iconSize: Icon.Default.prototype.options.iconSize,
+    iconAnchor: Icon.Default.prototype.options.iconAnchor,
+    popupAnchor: Icon.Default.prototype.options.popupAnchor,
+    shadowUrl: 'images/marker-shadow.png',
+    shadowSize: Icon.Default.prototype.options.shadowSize,
+    shadowAnchor: Icon.Default.prototype.options.shadowAnchor,
+  });
+  return (
+    <div>
+      {coords.map((x, index) => (
+        <Marker title='others-marker' icon={othersIcon} position={x} key={index} />
```

Comment:
Why use a hyphen here?

Reply:
do you want camelCase instead? Since it's an html attribute I would use hyphens, since we do it in classnames with tailwind too